all-in-one/README.md

```diff
@@ -0,0 +1,24 @@
+Steps to setup a local network
+==============================
+
+1. `make images`
+   If you don't want to build the images locally, you can skip this step.
+   Otherwise, you'll need the version number presented here if you don't use
+   latest in the next step:
+   `Built version: c6cd912-all-in-one-go1.10`
+
+2. `export VERSION=latest`
+   Export the version of the network you want to run. Latest should be ok, but
```

Comment:
Explain how to get the versions that you could use and what the versions mean?

Reply:
I think how to get the version is covered in the above step. The meaning isn't explained however.